ts/spi/provider_frame.ts

```diff
@@ -460,7 +460,7 @@ export class ProviderFrame {
       Promise<Credential[]> {
     // get all credentials across all domains; from this, we can filter down
     // to the set of credentials
-    let allCredentials = await this.credentialDataProvider.getAllCredentials();
+    let allCredentials = await this.credentialDataProvider.getAllHints(options);
```

Comment:
What is the notion of "AllHints" for the provider? It is confusing as the provider is actually not responsible for the actual hints displayed as OpenYolo perform some deduplications. What if the provider wants to show the user "duplicates" hints, for some reason? Maybe one hint has characteristics that the other one doesn't, and this is not captures in our method of de-duplicating.